test/coverage/datamodel/UUriValidatorTest.cpp

```diff
@@ -12,9 +12,13 @@
 #include <gtest/gtest.h>
 #include <up-cpp/datamodel/validator/UUri.h>
 
+#define AUTHORITY_NAME "test"
```

Comment:
I recommend moving this into the anonymous namespace and switching to a constexpr.

```suggestion
constexpr std::string_view AUTHORITY_NAME{"test"};
```

Reply:
Not exactly what you asked for wrt std::string_view, but similar.  The template functions really hated me using std::string_view and threw a bunch of compiler warnings "template argument deduction/substitution failed".  This change accomplishes what you were looking for I believe.